app/LoadedApp/LoadedApp.tsx

```diff
@@ -524,7 +524,10 @@ export class LoadedAppClass extends Component<LoadedAppClassProps, AppStateLoade
     }
     if (deepDiff(this.state.transactions, transactions)) {
       //console.log('fetch transactions');
-      this.setState({ transactions });
+      // set someUnconfirmed as well here when I know there is something new in transactions
+      const unconfirmed: number =
+        transactions.length > 0 ? transactions.filter((tx: TransactionType) => tx.confirmations === 0).length : 0;
+      this.setState({ transactions, someUnconfirmed: unconfirmed > 0 });
```

Comment:
does this check whether there are ANY unconfirmed notes?
the rest of the PR seems to be written as if this checks for ONLY transparent unconfirmed notes. if not, then it may cause some issues?

Reply:
Here, we are checking if there are any unconfirmed transactions. The problem is: I don't have a field called `spendableTransparent` and in some conditions you can create 2 or more transactions in a row, bacause the `transparentBal` is instantaneous even if the transaction is not mined yet.
We need one of two thing here:
- the balance for T don't change until the tx is mined/confirmed.
- a new field for `spendableTransparent`

Reply:
OK yes maybe we need spendableTransparent to make this behave nicely. Let's see how much work they think this will be @AloeareV @zancas @fluidvanadium